packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/CreateReceivables.tsx

```diff
@@ -121,6 +122,72 @@ const CreateReceivablesBase = ({
   }
 
   const className = 'Monite-CreateReceivable';
+  const handleCreateReceivable = (values: CreateReceivablesFormProps) => {
+    if (values.type !== 'invoice') {
+      throw new Error('`type` except `invoice` is not supported yet');
+    }
+
+    if (!actualCurrency) {
+      throw new Error('`actualCurrency` is not defined');
+    }
+
+    const billingAddressId = values.default_billing_address_id;
+    const counterpartBillingAddress = counterpartAddresses?.data?.find(
+      (address) => address.id === billingAddressId
+    );
+
+    if (!counterpartBillingAddress) {
+      throw new Error('`Billing address` is not provided');
+    }
```

Comment:
Let's instead of crashing and throw new Error provide for a user notification that something is wrong, otherwise this handleCreateReceivable will just crash

Reply:
Updated